src/snapred/backend/dao/state/CalibrantSample/CalibrantSample.py

```diff
@@ -27,3 +28,12 @@ class CalibrantSamples(BaseModel):
     @classmethod
     def set_datetime(cls, v: str) -> str:
         return v or str(datetime.datetime.now())
+
+    @field_validator("peakIntensityFractionThreshold", mode="before")
```

Comment:
That's cool, didn't know you could get pydantic to do validation before it validates the field for peakIntensityThreshold.